src/Contact/LocalRelationship/Exception/LocalRelationshipPersistenceException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Friendica\Contact\LocalRelationship\Exception;
+
+class LocalRelationshipPersistenceException extends \Exception
```

Comment:
I'm using a standard error code `500` for all persistence errors.

Maybe we have to think about backend error codes (`12345`) and frontend HTTP Error codes (`500`).

Reply:
I've started thinking about this, I like this idea but we would still need a mapping to HTTP error codes?

Reply:
yep

Reply:
Not in this PR though.